console/services/backup_service.py

```diff
@@ -145,6 +145,9 @@ def backup_group_apps(self, tenant, user, region, group_id, mode, note, force=Fa
             return backup_record_repo.create_backup_records(**record_data)
         except region_api.CallApiError as e:
             logger.exception(e)
+            if e.message["body"].get("msg",
```

Comment:
advice:
e.message["body"] => e.message.get('body', {})

Reply:
```region_api.CallApiError```This class has the body parameter when it is initialized